features/admin.extensions.v1/application-templates/templates/m2m-application/__tests__/quick-start.test.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { render, screen } from "@testing-library/react";
```

Comment:
We expose a modified render method for our unit tests in the test utils file. Let's use that function here by importing from `features/test-configs/utils`.

```suggestion
import { render, screen } from "../../../test-configs/utils";
```

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/5850/commits/5a6848d86668e0e09927f2a01bcdd5c94638196b